Opserver.Core/Data/Dashboard/Providers/WmiDataProvider.Polling.cs

```diff
@@ -44,7 +44,7 @@ public async Task<Node> PollNodeInfoAsync()
                         await PollStats().ConfigureAwait(false);
                     }
                 }
-                catch (COMException e)
+                catch (Exception e) when (e.InnerException is COMException)
```

Comment:
This seems to vary with earlier testing (you're failing in a new way) - instead of replacing here, we need to duplicate the `catch` for both :)